index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css"
+      integrity="sha384-b6lVK+yci+bfDmaY1u0zE8YYJt0TZxLEAFyYSLHId4xoVvsrQu3INevFKo+Xir8e"
+      crossorigin="anonymous"
+    />
+    <title>Mobile-First</title>
+  </head>
+  <body>
+    <main>
+    <section class="parent">
+      <div>
+        <nav>
+          <!--container 1-->
+          <div>
+            <h5>My Logo</h5>
+          </div>
+          <!--container 2-->
+          <div>
+            <i class="bi bi-list"></i>
+          </div>
+        </nav>
+        
+        <section class="bg-img">
+          <div>
+            <h1 class="name">I'm Maris<br/>Glad to see</br> you!</h1>
+            <div class="article">
+              I’m a software developer! I can help you<br>
+              build a product , feature or
+              website Look<br>
+              through some of my work and<br>
+              experience! If you like what
+              you see and<br>
+              have a project you need coded, don’t<br>
+              hesitate to contact me.
+            </div>
```

Comment:
- [ ] It would be better to use a `<p>` tag for the headline description for semantic reasons.